### DISCUSSION TITLE
FeedBack System

### DISCUSSION THREAD

**Discussion by sarantos92** [Vatsim-Scandinavia/controlcenter#388] [Ideas]
What about a FeedBack System for the controller?

FeedBack Sumbit > ATD and Controller involed in Review Notified

after the review from the training department if no action needed close case, if action needed notify controller to improve his services and give him a warning point, if he keeps doing the same thing on ex 5 warning the controller must go through a fast training refresh

**Comment by thor** [comment]
We've got a partial feedback system implemented in #664. I'm going to mark this one as closed, but feel free to create a feature request for the more specific, automated approach with training refreshes based on feedbacks. :)
